admin_numeric_filter/templates/admin/filter_numeric_slider.html

```diff
@@ -10,7 +10,7 @@
 
         <h3>{% blocktrans with filter_title=title %}By {{ filter_title }}{% endblocktrans %}</h3>
 
-        {% if choice.min and choice.max and choice.step %}
+        {% if choice.min or choice.min == 0 choice.min and choice.max and choice.step %}
```

Comment:
Hey, did you test that line? ;-) there is definitely too much choice.min

Reply:
Maybe we should do the same check with the max value as well. What do you think?

Reply:
:+1:  Please check now.

Reply:
What about this?

`{% if choice.min and choice.min is not None and choice.max is not None and choice.step %}`